app/assets/javascripts/application.js

```diff
@@ -102,6 +102,7 @@ var Search = {
     Search.observeFocus();
     Search.observeTextEntry();
     Search.observeResultsClicks();
+    Search.listenForKeys();
```

Comment:
A better name seems possible; this was the first thing that came to mind.

Reply:
Maybe `installKeyboardShortcuts()` is a more descriptive name?

Reply:
Stolen ;) thanks